packages/devtools-source-map/src/source-map.js

```diff
@@ -56,19 +56,8 @@ function _setSourceMapRoot(sourceMap, absSourceMapURL, source) {
     return;
   }
 
-  const base = path.dirname(
-    (absSourceMapURL.indexOf("data:") === 0 && source.url) ?
```

Comment:
Probably not, but this is a pathological situation: a source map claiming that the sources are at some `data:` URL, without providing the sources inline.  I can add the old special case back, but perhaps this should just be an error of some kind instead.